tools/crm_attribute.c

```diff
@@ -501,7 +458,7 @@ main(int argc, char **argv)
                                 options.attr_id, options.attr_name, &read_value, TRUE, NULL);
 
         if (rc == -ENXIO && options.attr_default) {
-            read_value = strdup(options.attr_default);
+            read_value = g_strdup(options.attr_default);
```

Comment:
read_value is char* and will be dynamically allocated in the usual way if read_attr_delegate() succeeds above, so it needs to keep using strdup() here. There's no problem strdup()'ing a gchar* (or g_strdup()'ing a char*).

Reply:
This two allocators thing is going to drive me crazy.

Reply:
Yep that's why we made a big effort years ago to eradicate all glib allocation of strings. Unfortunately we can't avoid it with GOption